pkgs/top-level/all-packages.nix

```diff
@@ -11307,10 +11307,10 @@ with pkgs;
 
   libsForQt59 = lib.makeScope qt59.newScope mkLibsForQt5;
 
-  qt510 = recurseIntoAttrs (makeOverridable
-    (import ../development/libraries/qt-5/5.10) {
+  qt511 = recurseIntoAttrs (makeOverridable
```

Comment:
If we remove each minor Qt version on each update, it doesn't seem there is much sense in keeping unstable attribute for that around.